lib/ansible/modules/packaging/language/pip.py

```diff
@@ -271,15 +273,22 @@ def _get_packages(module, pip, chdir):
     return (command, out, err)
 
 
-def _is_present(name, version, installed_pkgs, pkg_command):
+def _is_present(module, name, version, installed_pkgs, pkg_command):
     '''Return whether or not package is installed.'''
+    if version is not None:
+        if version[0].isdigit():
+            version = "==%s" % version
```

Comment:
This is repeated from above and could be extracted to a function or called earlier..